readme.txt

```diff
@@ -59,11 +59,14 @@ Any Bible references on your site will be displayed as links.
 
 1. Check to make sure you have activated the plugin from the Plugins page.
 1. Check to make sure you have JavaScript enabled in your web browser. 
-1. If you use a different font for your Bible references, such as italic text, go to the settings page at Settings > Faithlife Reftagger and make sure the script is allowed to search that kind of text.
+1. If you use a different font for your Bible references, such as italic text, go to the settings page at Settings > Logos Reftagger and make sure the script is allowed to search that kind of text.
 
 = See https://faithlife.com/products/reftagger/faq for more FAQs =
 
 == changelog ==
+2.4.5:
```

Comment:
We'll need to also update the version at the top of RefTagger.php

Reply:
Whoops, forgot to commit that file. Good catch